packages/agents-openai/src/openaiResponsesModel.ts

```diff
@@ -142,7 +147,12 @@ function converTool<_TContext = unknown>(
       return {
         tool: {
           type: 'file_search',
-          vector_store_ids: tool.providerData.vectorStoreId,
+          vector_store_ids:
+            tool.providerData.vectorStoreIds ||
+            // for backwards compatibility
+            (typeof tool.providerData.vectorStoreId === 'string'
```

Comment:
we already have vectorStoreId here but it should be vectorStoreIds (plural). allowing both here for backward compatibility and simple use cases w/ a single vector store.